app/views/idv/doc_auth/_spinner.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class='pl1 sm-col sm-col-3'>
```

Comment:
I added the alt tag I created for the spinner here and did a SLIM -> HTML conversion.